web_development_101/project_js_jquery.md

```diff
@@ -39,6 +39,7 @@ If you get totally stuck, solutions from other students are listed below.  I'd e
 *Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.*
 
 * Add your solution below this line!
+* [Ana Sommer's Solution](https://github.com/anasommer/Sketchie ) | [View in browser](https://anasommer.github.io/Sketchie/)
```

Comment:
Hi Ana,

You left a space between your [Ana Sommer's Solution] and (https://github.com/anasommer/Sketchie ) which means it won't display correctly. I've corrected it for you but just keep that in mind for future 👍